admin-dev/themes/default/template/header.tpl

```diff
@@ -230,19 +230,23 @@
 						</ul>
 					</li>
 				</ul>
-				{$quick_access_current_link_name = " - "|explode:$quick_access_current_link_name}
 				<script>
 					$(function() {
 						$('.ajax-quick-link').on('click', function(e){
 							e.preventDefault();
+
+							var method = $(this).data('method');
+							if(method == 'add')
+								var name = prompt("{l s='Please provide name for this shortcut' js=1}");
```

Comment:
Hi @kpodemski,

maybe you can add ```$quick_access_current_link_name``` as default value and check if name is empty do not fire ajax.

Best regards.